src/open_inwoner/conf/parts/case_status_notification.html

```diff
@@ -49,7 +49,7 @@ <h2>{{ type_description }}</h2>
     <tr>
         <td class="td-mail td-mail__bg-info td__padding-all td__info-link" colspan="6">
             <p>U hoeft niets te doen. Zodra er weer een update beschikbaar is ontvangt u hier een melding over. U kunt wel inloggen om uw aanvraag te bekijken.</p>
-            <p><strong><a href="{{ case_link }}" class="color--primary">Log in</a><span class="color--primary"> &rarr;</span></strong></p>
```

Comment:
The case_link is preferable, this gives you a redirect to the loginpage with a next-parameter to the detailview of the case